### ISSUE TITLE
no password requested on first run after reboot

### ISSUE THREAD

**Issue by wjaecklein** [nextcloud/android#7729]
### Steps to reproduce
1. Reboot Android device
2. Open NextCloud app
3. 

### Expected behaviour
- Tell us what should happen
- Should request password to be enteted (or fingerprint yo be read)

### Actual behaviour
- Tell us what happens
- The app just opens, giving access to all files without security check.

This does no longer happen upon second (or subsequent) access to app!

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:
10
Device model: 
Samsung Galaxy S10e
Stock or customized system:

Nextcloud app version:
3.14.1
Nextcloud server version:
Where do I find this information, please?
Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by tobiasKaminsky** [comment]
Have you pin protection or device credentials enabled?

If you have device credentials, as you unlock your phone on boot, there is a grace period of 10s, in which you do not need to enter password again.
So boot, wait ~1 min, and please see if it is still possible to open the app without password.

**Comment by wjaecklein** [comment]
My SIM-card is PIN protected, and my phone is password-locked (or fingerprint).
But no matter the delay between reboot and the first launch of keepass, there's no password requested upon *first* opening of the app. Any subsequent launch works as expected.

7 janvier 2021 14:38 "Tobias Kaminsky" <notifications@github.com (mailto:notifications@github.com?to=%22Tobias%20Kaminsky%22%20<notifications@github.com>)> a écrit:
	Have you pin protection or device credentials enabled?

	If you have device credentials, as you unlock your phone on boot, there is a grace period of 10s, in which you do not need to enter password again.
So boot, wait ~1 min, and please see if it is still possible to open the app without password.

**Comment by wjaecklein** [comment]
Sorry, my reply to your last question wasn’t recorded, as it seems.

I now have upgraded to Android Nextcloud version 3.15.0 (and Android 11, 
kernel 4.14.113-20607146, One UI 3.0).



The behavior does not change, no matter how long I wait between reboot and 
opening of the Nextcloud app, on *first* launch, I’m not requested to enter 
a password or to offer my fingerprint. Still, any subsequent launch of 
Nextcloud requests as expected my credentials.



Otherwise, I do not know what “device credentials” are or “pin protection” – 
but anyway, even a looooong wait between reboot and launch of Nextcloud is 
showing this unexpected behavior.

**Comment by tobiasKaminsky** [comment]
I found a bug and it is fixed via 3.15.1 RC1, can you test this via Play Store Beta program?

**Comment by tobiasKaminsky** [comment]
Fixed by #8026 

If it still happens, please re-open.
